geonode_mapstore_client/client/js/routes/Home.jsx

```diff
@@ -422,6 +435,19 @@ function Home({
             <div className="gn-main-home">
 
                 <div className="gn-container">
+                    <div className="gn-row">
+                        <div className="gn-grid-container">
+                            <ConnectedFeatureList
+                                query={query}
+                                location={location}
```

Comment:
We don't need to pass the location to check if the component should be rendered or not but we could add a props boolean to the Home route to disable/enable the FeaturedList (eg: disableFeatured). In this way we can add disableFeatured equal to true in the Search and Detail routes components
```js
{!disableFeatured && <ConnectedFeatureList
```